src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -399,46 +399,6 @@ protected void InitializeReferences()
             }
         }
 
-        private static Match? InternalPerformScan(bool quick, string input, int beginning, RegexRunner runner, ReadOnlySpan<char> span, bool returnNullIfQuick)
```

Comment:
Note that this method was only renamed (to RunInternal) and moved below but the contents were left the same.